csi.proto

```diff
@@ -172,6 +184,10 @@ message CreateVolumeRequest {
   // logged, etc.) by the CO.
   // This field is OPTIONAL.
   map<string, string> controller_create_secrets = 5;
+
+  // The ID of the source snapshot to create volume from.
+  // This field is OPTIONAL.
+  string snapshot_id = 6;
```

Comment:
I suggest renaming this field as `source_snapshot_id` so that it is less unambiguous, list like the response field on L301. If we support more source types in the future, the `source_*` prefix will help clarify what is being requested as the source.

Alternatively, adding a struct `source_snapshot_request` would allow more flexibility, in case we need more that the ID as Saad suggested in the meeting today.